Button/button.css

```diff
@@ -0,0 +1,30 @@
+.button{
+    padding: 0.5rem;
+    border: none;
+}
+.button-primary{
+    background-color: var(--primary-color);
+    color: white;
+    border-radius: 0.2rem;
+    margin: 0.5rem;
+}
+.button-icon{
+    background-color: var(--unread-color);
+    color: white;
+    border-radius: 0.2rem;
+    margin: 0.5rem;
```

Comment:
these styles are repeating can be added to the button class

Reply:
Done with these changes